src/components/JourneyComparison/JourneyComparison.tsx

```diff
@@ -127,8 +167,28 @@ const JourneyComparison = (props: ComparisonProps) => {
             <FrequencyComparison {...props} />
             <ComparisonRow
                 title="Your trip"
-                baseline={<JourneyTimeline segments={baseline.segments} />}
-                enhanced={<JourneyTimeline segments={enhanced.segments} />}
+                baseline={
+                    <JourneyTimeline
+                        segments={
+                            !departAfter ||
```

Comment:
I think we could avoid these checks, too, by removing the initial `transfer` segment on the server.